panel/README.md

```diff
@@ -0,0 +1,29 @@
+# Panel
+
```

Comment:
Can we add installation instructions here?

Reply:
The original idea was to put it there but there is a PR to restructure the codebase that will make it redundant so I purposefully left it out

good catch though!